src/content/core/okteto-variables.mdx

```diff
@@ -107,6 +176,102 @@ destroy:
       terraform apply -input=false -var "bucket_name=${S3_BUCKET_NAME}" -var "queue_name=$resourceName" -var "region=$region" -auto-approve --destroy
 ```
 
+
+
+## Setting Okteto Variables
+
+This section will show a brief example of how to set each type of Okteto Variable.
+
+### The --var flag
+
+The `--var` flag is useful to specify ad-hoc variables when running a command like [deploy](/docs/reference/okteto-cli#deploy) or [preview deploy](/docs/reference/okteto-cli#deploy-3)
+
+Example:
+
+```console
+$ okteto deploy --var PORT=4000
+```
+
+```yaml
+deploy:
+  image: "node:${TAG}"
+  commands:
+    - name: deploy
+      command: NODE_ENV=${ENV} make deploy
+```
+- Variables cab be used in `deploy`, `destroy` and `test` commands in the Okteto manifest
+
+### Catalog
+
+When you deploy an application from the Okteto Dashboard or if your organization is using the Okteto Catalog, you can define project-specific variables. See [Catalog Variables](/docs/admin/catalog/#add-environment-variables) for more information.
+
+### Local Environment Variables
+
+Similar to the `--var` flag, local environment variables are useful when you need to specify ad-hoc variables when running a command like [deploy](/docs/reference/okteto-cli#deploy) or [preview deploy](/docs/reference/okteto-cli#deploy-3)
+
+Example:
+
+```console
+$ PORT=4000 okteto deploy
+```
+
+
+### The .env file
+
+You can set default values for any environment variables referenced in the okteto manifest in an environment file named `.env`.
+The `.env` file is placed at the same folder than the okteto manifest.
+
+For example:
+
+```console
+$ cat .env
+TAG=v1.5
+```
+
+```console
+$ cat okteto.yaml
+...
+    image: "node:${TAG}"
+...
+```
+
+When you run any Okteto CLI command, `image` will be `node:v1.5`.
+
+> Values in the shell take precedence over those specified in the `.env` file.
+
+
+### User and Admin variables
+
+You can create and delete your Okteto Variables from the `Variables` section in the `Settings` view of the Okteto dashboard:
```

Comment:
We might need to rephrase this and update screenshots according to this PR here: https://github.com/okteto/app/pull/8040 for Admin Variables